modules/sdk/gradle-plugins-workspace/src/gradleTest/clientExtensionValidateTask/testShouldThrowExceptionWhenInvalidTokenDefinitionFile.gradle

```diff
@@ -0,0 +1,22 @@
+import com.liferay.gradle.plugins.workspace.configurator.ClientExtensionProjectConfigurator
+
+task runGradleTest
+
+runGradleTest {
+	doLast {
+		String validateClientExtensionsTaskName = ClientExtensionProjectConfigurator.VALIDATE_CLIENT_EXTENSIONS_TASK_NAME
+
+		Project project = project(":client-extension:any-theme-css-project-invalid-token-definition-file")
+
+		Task validateTask = project.tasks[validateClientExtensionsTaskName]
```

Comment:
Source formatter seems to not fixing this, but there should be an empty line here.

We have [this convention](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DBlank-lines-separate-statements) but I remember one saying there should be an empty line between declaring a variable and using it